src/Tribe/Events/Attendee_List_Display.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Tribe\Tickets\Events;
+
+use Exception;
+use InvalidArgumentException;
+use Tribe__Tickets_Plus__Attendees_List;
+use WP_Post;
+
+class Attendee_List_Display {
+
+	private static $attendee_list_by_shortcode = 'tribe_tickets_attendee_list_triggered_by_shortcode';
+
+	/**
+	 * Check if given event is hiding the attendees list.
+	 *
+	 * @param int|WP_Post $post
+	 *
+	 * @throws InvalidArgumentException
+	 *
```

Comment:
:no_entry_sign: Comment missing for @throws tag in function comment

<sub>hash: ac8b4e1d34cd7de1750bec78c5ba213b</sub>